composer.json

```diff
@@ -14,12 +14,12 @@
     {"name" : "Aurélien FOUCRET", "email": "aurelien.foucret@elastic.co"}
   ],
   "require" : {
-    "php" : "^5.6|^7.0",
+    "php" : "^5.6|^7.0|^8.0",
```

Comment:
Wouldn't `>=7.0` make more sense here? It would properly deprecate 5.6, as it should be, but still allow the EOL 7 versions of PHP, and for the time being, work with 8.